src/platform/Linux/CHIPBluezHelper.cpp

```diff
@@ -650,29 +638,27 @@ static gboolean BluezIsCharOnService(BluezGattCharacteristic1 * aChar, BluezGatt
                                                                                                                             : FALSE;
 }
 
-static uint16_t BluezUUIDStringToShortServiceID(const char * aService)
+static bool ParseBleUuid(const char * str, ChipBleUUID & uuid)
```

Comment:
is this re-useable across platforms?   Should we move it up?  Can you include tests for it?

Reply:
yeah, good idea!

Reply:
I moved it to BleUUID.h and added UTs.